ui/cypress/integration/node/nodetabs.spec.js

```diff
@@ -0,0 +1,121 @@
+beforeEach(() => {
+  cy.setupMocks();
+  cy.login();
+});
+
+// Navigation tests
+describe('Overview tab in master-0 node page', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.visit('/nodes/master-0/overview');
+    cy.route('api/kubernetes/api/v1/nodes', 'fixture:kubernetes/nodes.json');
+    cy.window()
+      .its('__history__')
+      .then((history) => {
+        cy.stub(history, 'push').as('historyPush');
+      });
+  });
+
+  const tabs = ['alerts', 'metrics', 'volumes', 'pods', 'details'];
+  tabs.map((tab) => {
+    it(`brings me to the [${tab}] tab`, () => {
```

Comment:
We shouldn't use square brackets in test names - if the test fails, the generated screenshot will have square brackets in its name and we cannot upload such files to artifacts today (discovered this when debugging CI issues).

Reply:
Thanks for letting me know this point!